tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-product-loader-test.php

```
@@ -0,0 +1,361 @@
+		$variable_ids = $this->create_variable_products( 2 );
+
+		$loader = new WC_Stripe_Agentic_Commerce_Product_Loader();
+		$args   = $this->paginated_args( $simple_ids, $variable_ids, 1, 2 );
+
+		$result = $loader->get_products( $args );
+
+		$this->assertInstanceOf( \stdClass::class, $result );
+		$this->assertSame( array_slice( $simple_ids, 0, 2 ), $this->ids( $result->products ) );
+		$this->assertSame( 4 + 2, $result->total );
+		$this->assertSame( 3, $result->max_num_pages );
+	}
```

PHPMD (2.15.0)

[error] The method test_paginated_page_one_parent_fully_satisfies_returns_parent_only is not named in camelCase. (undefined)

(CamelCaseMethodName)